src/data/python-quiz.ts

```diff
@@ -1,4 +1,15 @@
 const pythonQuiz = [
+    {
+    Question: 
+      "In Python, word1 ='free'and word2=' ode camp' then what will be the value of print(word1+word2)?",
```

Comment:
It looks you are missing a c for code based on the answers you supplied. 

```suggestion
      "In Python, if we have word1 ='free' and word2=' code camp', then what is the result here: print(word1+word2)?",
```